src/GeneralStateTestsFiller/stReturnDataTest/returndatacopy_0_0_following_successful_createFiller.json

```diff
@@ -0,0 +1,74 @@
+{
+    "returndatacopy_0_0_following_successful_create" : {
+        "env" : {
+            "currentCoinbase" : "0x2adc25665018aa1fe0e6bc666dac8fc2697ff9ba",
+            "currentDifficulty" : "0x20000",
+            "currentGasLimit" : "0x0a00000000",
+            "currentNumber" : "0x01",
+            "currentTimestamp" : "0x03e8",
+            "previousHash" : "0x5e20a0453cecd065ea59c37ac63e079ee08998b6045136a8ce6635c7912ec0b6"
+        },
+        "expect" : [
+            {   "indexes" : {
+                   "data" : -1,
+                   "gas" : -1,
+                   "value" : -1
+                },
+                "network" : ["Frontier", "Homestead", "EIP150", "EIP158"],
+                "result" : {
+                    "0x0f572e5295c57f15886f9b263e2f6d2d6c7b5ec6" : {
+                        "storage" : {
+                            "0x00" : "0x01"
+                        }
+                    }
+                }
+            },
+            {   "indexes" : {
+                   "data" : -1,
+                   "gas" : -1,
+                   "value" : -1
+                },
+                "network" : ["Metropolis"],
+                "result" : {
+                    "0x0f572e5295c57f15886f9b263e2f6d2d6c7b5ec6" : {
+                        "storage" : {
+                            "0x00" : "0x00"
+                        }
+                    }
+                }
+            }
+        ],
+        "pre" : {
+            "0x0f572e5295c57f15886f9b263e2f6d2d6c7b5ec6" : {
+                "balance" : "0x00",
+                "code" : "{ (create (STOP)) (RETURNDATACOPY 0 0 0) (SSTORE 0 (MLOAD 0)) }",
```

Comment:
you could use sstore  instead of stop and see the new contract in the post state